pandas/core/groupby.py

```diff
@@ -1933,6 +1933,23 @@ def shift(self, periods=1, freq=None, axis=0):
                                            self.grouper, needs_ngroups=True,
                                            periods=periods)
 
+    @Substitution(name='groupby')
+    @Appender(_doc_template)
+    def pct_change(self, periods=1, fill_method='pad', limit=None, freq=None,
+                   axis=0):
+        """Calcuate pct_change of each value to previous entry in group"""
+        if freq is not None or axis != 0:
```

Comment:
I copied this from the `shift` implementation but I'm considering moving this into a decorator or pushing into the `_get_cythonized_result` methods as it is useful for quite a few functions